### PR TITLE
Jordeen/task deletion

### PR DESCRIPTION
**Pull request by jodreen**
task deletion works!!!

### CODE REVIEW

**Comment by nishdesai** [comment]
TYBG

**Comment by nishdesai** [comment]
*TYBJ

File: watershed-iOS/Watershed/Watershed/WPEditTaskViewController.m [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+//    
+//    // Don't request the list of sites, because it is already called in the ViewController's viewWillAppear
+//    __weak __typeof(self)weakSelf = self;
+//    [[WPNetworkingManager sharedManager] editTaskWithTask:
+//                                               parameters:[[NSMutableDictionary alloc] init]
+//                                                  success:^(WPTask *task) {
+//        __strong __typeof(weakSelf)strongSelf = weakSelf;
+//        [strongSelf dismissSelf];
+//    }];
+//}
```
Comment (nishdesai) [inline watershed-iOS/Watershed/Watershed/WPEditTaskViewController.m]: Should this be here? Not sure why we would need to request sites in the task editing view controller.

File: watershed-iOS/Watershed/Watershed/WPEditTaskViewController.m [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    self.siteField.text = self.selectedMiniSite.name;
+    self.descriptionView.text = self.task.taskDescription;
+    self.urgentSwitch.selected = self.task.urgent;
+}
+
+#pragma mark - Setter Methods
+
+- (void)setTask:(WPTask *)task{
+    _task = task;
+//    [self preloadFields];
```
Comment (nishdesai) [inline watershed-iOS/Watershed/Watershed/WPEditTaskViewController.m]: also this

File: watershed-iOS/Watershed/Watershed/WPTaskViewController.m [hunk 3]
```diff
@@ -37,30 +39,75 @@ - (void)didReceiveMemoryWarning {
...
+                                                  otherButtonTitles:@"Yes",nil];
+    [confirmDelete show];
+}
+
+- (void)editTask {
+    WPEditTaskViewController *editTaskViewController = [[WPEditTaskViewController alloc] init];
+    [editTaskViewController setTask:self.task];
+//    [editTaskNavController.navigationBar setBackgroundColor:[UIColor whiteColor]];
+//    [editTaskNavController.navigationBar setBarTintColor:[UIColor whiteColor]];
+//    [editTaskNavController.navigationBar setTitleTextAttributes:@{NSForegroundColorAttributeName : [UIColor blackColor]}];
```
Comment (nishdesai) [inline watershed-iOS/Watershed/Watershed/WPTaskViewController.m]: Does the view still look fine without this styling? Also we should make sure our views remain compatible with iOS 7 (we had a problem with that before when we were releasing the beta)

**Comment by jodreen** [comment]
Shoot these commits are supposed to be on a different branch sorry i'll fix it

it's still in progress

**Comment by nishdesai** [comment]
Gotcha. So this isn't ready to merge?

**Comment by jodreen** [comment]
it is now!! I moved all the task editing commits to another branch, so this should be good to go

**Pull request merged**